content/en/docs/components/pipelines/legacy-v1/introduction.md

```diff
@@ -209,25 +209,25 @@ Prediction results:
 
 <img src="/docs/images/predict.png" 
   alt="Prediction output"
-  class="mt-3 mb-3 p-3 border border-info rounded">
+  class="mt-3 mb-3 border rounded">
 
 Confusion matrix:
 
 <img src="/docs/images/cm.png" 
   alt="Confusion matrix"
-  class="mt-3 mb-3 p-3 border border-info rounded">
+  class="mt-3 mb-3 border rounded">
 
 Receiver operating characteristics (ROC) curve:
 
 <img src="/docs/images/roc.png" 
   alt="ROC"
-  class="mt-3 mb-3 p-3 border border-info rounded">
+  class="mt-3 mb-3 border rounded">
 
 ## Architectural overview
 
 <img src="/docs/images/pipelines-architecture.png" 
```

Comment:
![pipelines-dark](https://github.com/user-attachments/assets/9e6633ac-400d-43ea-b078-5798cace2829)
In the diagram there are few words that are not visible due to the dark background.

Reply:
This has been fixed.